pkg/controller/plan/validation.go

```diff
@@ -629,6 +638,15 @@ func (r *Reconciler) validateVM(plan *api.Plan) error {
 				setOfTargetName[vm.TargetName] = true
 			}
 		}
+		// check for supported OVA source
+		if ova, ok := v.(*ova.VM); ok {
+			for _, concern := range ova.Concerns {
+				// match label from ova/export_source.rego
+				if concern.Label == "Unsupported OVA source" {
```

Comment:
we now have ids :tada: , we can test by id witch is nicer then by label:

"id": "ova.source.unsupported",